dashboard/USER_GUIDE.md

```diff
@@ -13,36 +13,23 @@ at least one test has failed against the top of tree and flutter/flutter is
 currently not accepting new commits. Check on Discord [#tree-status](https://discord.com/channels/608014603317936148/613398423093116959)
 for discussion about tree closures.
 
-Tree closures occur when the latests results for a test have returned as
-failing. Logs are found by clicking a task box then clicking "Download Logs."
+Tree closures occur when the latest results for a test have returned as
+failing. Logs are found by clicking a task box then clicking "view logs."
 
-### Why am I not able to download logs?
-
-Ensure that you are signed in to the app (top right).
-
-Reach out on Discord [#hackers-infra](https://discord.com/channels/608014603317936148/608021351567065092)
-to be added as an AllowListedAccount to Cocoon.
-
-When the DeviceLab migrates to LUCI, logs will be available without sign in.
 
 ### Why is a task stuck on "new task" status?
 
 The dashboard aggregates build results from multiple build environments,
-including Cirrus, LUCI, and DeviceLab. DeviceLab and Cirrus test every commit
-that goes into the `master` branch. However, LUCI may skip commits when they
-come in too fast.
+including Cirrus, LUCI, and DeviceLab. Due to limited capacity to run tests on
```

Comment:
DeviceLab is now also LUCI.

Reply:
I understand, but I think it takes away the the complexity here (we have multiple ways of validating tests, and each of them can be different). I could split this into LUCI (VM) and LUCI (DeviceLab), but that seems verbose. WDYT?

Reply:
Actually we do not have any cirrus on post submit tests for different repos. Could we rephrase it to something like below? 
```
The dashboard aggregates build results from LUCI, including hostonly and devicelab.
```

Reply:
My understanding is we still want to support cirrus for post-submit, but we don't actively use it today - See https://cs.opensource.google/flutter/cocoon/+/main:app_dart/lib/src/request_handlers/refresh_cirrus_status.dart and there's quite a few other places in the code that support it

Reply:
These are for presubmit tests. @godofredoc is it expected we will not support cirrus post-submit tests in our CI?

Reply:
Packages and plugins are still using cirrus for post-submit and they have no plans on migrating everything to LUCI. If we can convince them to move all the post-submit testing to LUCI then we can safely assume cirrus will be running only on presubmit.

Reply:
Thanks Godofredo. With that said, I'm going to leave this as is. SG Keyong?

Reply:
SGTM!